# Translations template for nova.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the nova project.
#
# Translators:
# jaekwon.park <jaekwon.park@rockplace.co.kr>, 2014
# Andreas Jaeger <jaegerandi@gmail.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: nova 14.0.0.0b3.dev750\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2016-08-25 06:15+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2016-03-22 09:56+0000\n"
"Last-Translator: SeYeon Lee <sy_lee@kr.ibm.com>\n"
"Language: ko-KR\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.3\n"
"Language-Team: Korean (South Korea)\n"

msgid "\"Look for the VDIs failed"
msgstr "\"VDI 검색 실패"

#, python-format
msgid ""
"%(base)s virtual size %(base_size)s larger than flavor root disk size "
"%(size)s"
msgstr ""
"%(base)s 가상 크기 %(base_size)s이(가) flavor 루트 디스크 크기 %(size)s보다 "
"큼"

#, python-format
msgid ""
"%(scheme)s is registered as a module twice. %(module_name)s is not being "
"used."
msgstr ""
"%(scheme)s이(가) 모듈로 두 번 등록되었습니다. %(module_name)s을(를) 사용 중이"
"지 않습니다."

#, python-format
msgid ""
"Aggregate %(aggregate_id)s: unrecoverable state during operation on %(host)s"
msgstr ""
"집계 %(aggregate_id)s: %(host)s에 대한 오퍼레이션 중에 회복 불가능 상태가 됨"

msgid "An error occurred while deallocating network."
msgstr "네트워크를 할당 해제하는 동안 오류가 발생했습니다."

msgid "An error occurred while refreshing the network cache."
msgstr "네트워크 캐시를 새로 고치는 중에 오류가 발생했습니다."

#, python-format
msgid "Attach volume failed to %(instance_name)s: %(exn)s"
msgstr "%(instance_name)s에 볼륨 연결 실패: %(exn)s"

#, python-format
msgid "Attaching PCI devices %(dev)s to %(dom)s failed."
msgstr "PCI 디바이스 %(dev)s을(를) %(dom)s에 연결하는 데 실패했습니다."

#, python-format
msgid "Attaching network adapter failed. Exception: %s"
msgstr "네트워크 어댑터에 연결하는 데 실패했습니다. 예외: %s"

msgid "Cannot cleanup migration files"
msgstr "마이그레이션 파일을 정리할 수 없음"

#, python-format
msgid ""
"Cannot log in target %(target_iqn)s. Unsupported iSCSI authentication "
"method: %(auth_method)s."
msgstr ""
"대상 %(target_iqn)s을(를) 로그인할 수 없습니다. 지원되지 않는 iSCSI 인증 메소"
"드: %(auth_method)s."

#, python-format
msgid "Cannot reboot instance: %s"
msgstr "인스턴스를 재부팅할 수 없음: %s"

#, python-format
msgid "Caught error: %s"
msgstr "오류 발견: %s"

msgid "Compute driver option required, but not specified"
msgstr "계산 드라이버 옵션이 필수이지만 지정되지 않았음"

#, python-format
msgid ""
"Connection between volume %(vol)s and host %(host)s might have succeeded, "
"but attempt to terminate connection has failed. Validate the connection and "
"determine if manual cleanup is needed. Error: %(msg)s Code: %(code)s."
msgstr ""
"%(vol)s 볼륨과 %(host)s 호스트 사이의 연결에 성공했지만, 연결을 종료하려는 시"
"도에 실패했습니다. 연결을 확인하고 수동 정리가 필요한지 판별하십시오. 오류: "
"%(msg)s 코드: %(code)s."

#, python-format
msgid "Connection to libvirt failed: %s"
msgstr "libvirt 연결 실패: %s"

#, python-format
msgid "Could not bind to %(host)s:%(port)s"
msgstr "%(host)s:%(port)s에 바인드할 수 없음"

msgid "Could not decode scheduler options"
msgstr "스케줄러 옵션을 디코딩할 수 없음"

#, python-format
msgid ""
"Could not disconnect encrypted volume %(volume)s. If dm-crypt device is "
"still active it will have to be destroyed manually for cleanup to succeed."
msgstr ""
"암호화된 볼륨 %(volume)s의 연결을 끊을 수 없습니다. dm-crypt 디바이스가 활성 "
"상태인 동안 수동으로 영구 삭제해야 정리에 성공합니다."

#, python-format
msgid "Could not remove tmpdir: %s"
msgstr "tmpdir을 제거할 수 없음: %s"

#, python-format
msgid "Could not start encryption for disk %(device)s: %(exception)s"
msgstr "%(device)s 디스크의 암호화를 시작할 수 없음: %(exception)s"

#, python-format
msgid "Could not stat scheduler options file %(filename)s"
msgstr "스케줄러 옵션 파일 %(filename)s의 통계를 낼 수 없음"

msgid "Couldn't communicate with any cells"
msgstr "셀과 통신할 수 없음"

#, python-format
msgid "Couldn't communicate with cell '%s'"
msgstr "'%s' 셀과 통신할 수 없음"

#, python-format
msgid "Couldn't lookup app: %s"
msgstr "앱을 검색할 수 없음: %s"

#, python-format
msgid "Couldn't unmount the GlusterFS share %s"
msgstr "GlusterFS 공유 %s의 마운트를 해제할 수 없음"

#, python-format
msgid "Couldn't unmount the NFS share %s"
msgstr "NFS 공유 %s의 마운트를 해제할 수 없음"

#, python-format
msgid "Couldn't unmount the Quobyte Volume at %s"
msgstr "%s의 Quobyte 볼륨을 마운트 해제할 수 없음"

#, python-format
msgid "Couldn't unmount the share %s"
msgstr "공유 %s의 마운트를 해제할 수 없음"

#, python-format
msgid "Creating config drive failed with error: %s"
msgstr "구성 드라이브 작성 실패. 오류: %s"

msgid "DB error"
msgstr "DB 오류"

msgid ""
"Data migrations for PciDevice are not safe, likely because not all services "
"that access the DB directly are updated to the latest version"
msgstr ""
"DB에 직접 액세스하는 일부 서비스가 최신 버전으로 업데이트되지 않을 가능성이 "
"크므로 PciDevice의 데이터 마이그레이션이 안전하지 않음"

msgid "Destroy VM failed"
msgstr "VM 영구 삭제 실패"

msgid "Destroy instance failed"
msgstr "인스턴스 영구 삭제 실패"

#, python-format
msgid "Detaching network adapter failed. Exception: %s"
msgstr "네트워크 어댑터에서 연결을 해제하는 데 실패했습니다. 예외: %s"

#, python-format
msgid ""
"Download handler '%(handler)s' raised an exception, falling back to default "
"handler '%(default_handler)s'"
msgstr ""
"'%(handler)s' 핸들러를 다운로드하면 예외 발생, 기본 핸들러 "
"'%(default_handler)s'(으)로 돌아감"

msgid "Download image error"
msgstr "이미지 다운로드 오류"

#, python-format
msgid "Driver failed to attach volume %(volume_id)s at %(mountpoint)s"
msgstr "드라이버가 %(mountpoint)s에서 %(volume_id)s 볼륨을 연결하는 데 실패"

msgid "Environment variable 'NETWORK_ID' must be set."
msgstr "환경 변수 'NETWORK_ID'가 설정되어야 합니다."

#, python-format
msgid ""
"Error contacting glance server '%(server)s' for '%(method)s', %(extra)s."
msgstr ""
"'%(method)s', %(extra)s의 Glance 서버 '%(server)s'에 접속하는 중 오류가 발생"
"했습니다."

#, python-format
msgid "Error defining a domain with XML: %s"
msgstr "XML로 도메인을 정의하는 중 오류 발생: %s"

#, python-format
msgid "Error defining a secret with XML: %s"
msgstr "XML로 시크릿 정의 오류: %s"

#, python-format
msgid "Error deleting conntrack entries for %s"
msgstr "%s의 conntrack 항목을 삭제하는 중 오류 발생"

#, python-format
msgid "Error deploying instance %(instance)s on baremetal node %(node)s."
msgstr ""
"베어메탈(baremetal) 노드 %(node)s에 %(instance)s 인스턴스를 배치하는 중에 오"
"류가 발생했습니다."

#, python-format
msgid "Error enabling hairpin mode with XML: %s"
msgstr "XML로 hairpin 모드를 사용하는 중 오류 발생: %s"

#, python-format
msgid "Error from last host: %(last_host)s (node %(last_node)s): %(exc)s"
msgstr ""
"마지막 호스트에서 오류 발생: %(last_host)s (노드 %(last_node)s): %(exc)s"

#, python-format
msgid "Error from libvirt during destroy. Code=%(errcode)s Error=%(e)s"
msgstr "영구 삭제 중 libvirt에서 오류 발생. 코드=%(errcode)s 오류=%(e)s"

#, python-format
msgid "Error from libvirt during undefine. Code=%(errcode)s Error=%(e)s"
msgstr "undefine 중 libvirt에서 오류 발생. 코드=%(errcode)s 오류=%(e)s"

#, python-format
msgid "Error from libvirt during unfilter. Code=%(errcode)s Error=%(e)s"
msgstr "unfilter 중 libvirt에서 오류 발생. 코드=%(errcode)s 오류=%(e)s"

#, python-format
msgid ""
"Error from libvirt while injecting an NMI to %(instance_uuid)s: [Error Code "
"%(error_code)s] %(ex)s"
msgstr ""
"NMI를 %(instance_uuid)s에 삽입하는 동안 libvirt에서 오류 발생: [오류 코드 "
"%(error_code)s] %(ex)s"

#, python-format
msgid "Error injecting data into image %(img_id)s (%(e)s)"
msgstr "%(img_id)s(%(e)s) 이미지에 데이터 삽입 오류"

#, python-format
msgid "Error instantiating %(provider)s: %(exception)s"
msgstr "%(provider)s 인스턴스화 오류: %(exception)s"

#, python-format
msgid "Error launching a defined domain with XML: %s"
msgstr "XML로 정의된 도메인을 실행하는 중 오류 발생: %s"

msgid "Error locating next hop for message"
msgstr "메시지의 다음 hop을 찾는 중 오류 발생"

msgid "Error locating next hops for message"
msgstr "메시지의 다음 hop을 찾는 중 오류 발생"

msgid ""
"Error occurred during volume_snapshot_create, sending error status to Cinder."
msgstr ""
"volume_snapshot_create 중에 오류가 발생하여 오류 상태를 Cinder에 보냅니다."

msgid ""
"Error occurred during volume_snapshot_delete, sending error status to Cinder."
msgstr ""
"volume_snapshot_delete 중에 오류가 발생하여 오류 상태를 Cinder에 보냅니다."

#, python-format
msgid "Error on '%(path)s' while checking direct I/O: '%(ex)s'"
msgstr "직접 I/O를 검사하는 중에 '%(path)s'에서 오류 발생: '%(ex)s'"

#, python-format
msgid ""
"Error preparing deploy for instance %(instance)s on baremetal node %(node)s."
msgstr ""
"베어메탈(baremetal) 노드 %(node)s에 %(instance)s 인스턴스를 배치하기 위해 준"
"비하는 중 오류가 발생했습니다."

msgid "Error processing message locally"
msgstr "메시지를 로컬로 처리하는 중 오류 발생"

#, python-format
msgid "Error releasing DHCP for IP %(address)s with MAC %(mac_address)s"
msgstr "MAC이 %(mac_address)s인 IP %(address)s의 DHCP를 해제하는 중 오류 발생"

#, python-format
msgid "Error scheduling instances %(instance_uuids)s"
msgstr "%(instance_uuids)s 인스턴스 스케줄 작성 오류"

msgid "Error sending message to next hops."
msgstr "다음 hop에 메시지를 보내는 중 오류 발생"

#, python-format
msgid "Error starting xvp: %s"
msgstr "xvp 시작 오류: %s"

msgid "Error trying to Rescue Instance"
msgstr "인스턴스를 복구하는 중 오류 발생"

msgid "Error trying to reschedule"
msgstr "다시 스케줄하려는 중에 오류 발생"

#, python-format
msgid "Error updating resources for node %(node)s."
msgstr "%(node)s 노드의 리소스를 업로드하는 중 오류가 발생했습니다."

msgid "Error waiting for responses from neighbor cells"
msgstr "이웃 셀에서 응답을 기다리는 중 오류 발생"

#, python-format
msgid "Error while trying to clean up image %s"
msgstr "%s 이미지 정리 중 오류 발생"

#, python-format
msgid "Error writing to %(path)s: %(exception)s"
msgstr "%(path)s에 쓰기 오류: %(exception)s"

#, python-format
msgid "Error: %s"
msgstr "오류: %s"

#, python-format
msgid "Exception dispatching event %(event)s: %(ex)s"
msgstr "%(event)s 이벤트를 디스패치하는 중에 예외 발생: %(ex)s"

#, python-format
msgid "Exception handling resource: %s"
msgstr "자원 처리 예외: %s"

msgid "Exception in string format operation"
msgstr "문자열 형식화 오퍼레이션의 예외"

msgid "Exception loading extension"
msgstr "확장을 로드하는 중에 예외 발생"

#, python-format
msgid "Exception running %(name)s %(type)s-hook: %(obj)s"
msgstr "%(name)s %(type)s-hook을(를) 실행하는 중 예외 발생: %(obj)s"

msgid "Exception while creating table."
msgstr "테이블 작성 중에 예외가 발생했습니다. "

#, python-format
msgid "Exception while waiting completion of volume snapshots: %s"
msgstr "볼륨 스냅샷이 완료될 때까지 기다리는 중에 예외 발생: %s"

#, python-format
msgid "Extending virtual disk failed with error: %s"
msgstr "다음 오류로 인해 가상 디스크를 확장하는 데 실패: %s"

#, python-format
msgid "Failed removing bridge device: '%s'"
msgstr "브릿지 디바이스를 제거하는 데 실패: '%s'"

#, python-format
msgid "Failed removing net device: '%s'"
msgstr "네트 디바이스를 제거하는 데 실패: '%s'"

msgid "Failed storing info cache"
msgstr "정보 캐시를 저장하지 못했음"

msgid ""
"Failed to allocate PCI devices for instance. Unassigning devices back to "
"pools. This should not happen, since the scheduler should have accurate "
"information, and allocation during claims is controlled via a hold on the "
"compute node semaphore"
msgstr ""
"인스턴스의 PCI 디바이스를 할당하는 데 실패했습니다. 디바이스의 할당을 해제하"
"여 풀로 다시 보냅니다. 스케줄러에 정확한 정보가 있어야 하며 청구 중 할당은 컴"
"퓨트 노드 세마포어의 보류를 통해 제어되므로 이 문제는 발생하지 않아야 합니다."

msgid "Failed to allocate network(s)"
msgstr "네트워크를 할당하는 데 실패"

#, python-format
msgid "Failed to attach %(volume_id)s at %(mountpoint)s"
msgstr "%(mountpoint)s에서 %(volume_id)s을(를) 연결하는 데 실패"

#, python-format
msgid "Failed to attach volume at mountpoint: %s"
msgstr "마운트 포인트에 볼륨을 연결하는 데 실패: %s"

#, python-format
msgid "Failed to build configdrive: %s"
msgstr "configdrive를 빌드하는 데 실패: %s"

#, python-format
msgid "Failed to cancel migration %s"
msgstr "%s 마이그레이션을 취소하는 데 실패"

#, python-format
msgid "Failed to change vm state of %(instance_name)s to %(req_state)s"
msgstr "vm 상태를 %(instance_name)s에서 %(req_state)s(으)로 변경하는 데 실패"

msgid "Failed to check if instance shared"
msgstr "인스턴스 공유 여부를 확인하는 데 실패"

#, python-format
msgid "Failed to cleanup directory %(target)s: %(e)s"
msgstr "디렉토리 %(target)s 정리 실패: %(e)s"

msgid "Failed to cleanup snapshot."
msgstr "스냅샷을 정리하는 데 실패했습니다."

#, python-format
msgid "Failed to commit reservations %s"
msgstr "%s 예약을 커밋하는 데 실패"

msgid "Failed to complete a deletion"
msgstr "삭제를 완료하는 데 실패"

#, python-format
msgid "Failed to connect to volume %(volume_id)s with volume at %(mountpoint)s"
msgstr "%(volume_id)s 볼륨을 %(mountpoint)s의 볼륨과 연결하는 데 실패"

#, python-format
msgid ""
"Failed to copy cached image %(source)s to %(dest)s for resize: %(error)s"
msgstr ""
"크기를 조정하기 위해 캐시된 이미지 %(source)s을(를) %(dest)s(으)로 복사하는 "
"데 실패: %(error)s"

msgid "Failed to deallocate networks"
msgstr "네트워크 할당 해제 실패"

#, python-format
msgid "Failed to decrypt %(image_location)s to %(image_path)s"
msgstr "%(image_location)s을(를) %(image_path)s(으)로 복호화하지 못했음"

#, python-format
msgid "Failed to destroy instance: %s"
msgstr "인스턴스 영구 삭제 실패: %s"

#, python-format
msgid "Failed to detach %(device_name)s. Exception: %(exc)s"
msgstr "%(device_name)s의 연결을 해제하는 데 실패했습니다. 예외: %(exc)s"

#, python-format
msgid "Failed to detach volume %(volume_id)s from %(mp)s"
msgstr "%(mp)s에서 %(volume_id)s 볼륨을 분리하지 못했음"

#, python-format
msgid "Failed to download %(image_location)s to %(image_path)s"
msgstr "%(image_location)s을(를) %(image_path)s(으)로 다운로드하지 못했음"

#, python-format
msgid ""
"Failed to extract compute RPC version from service history because I am too "
"old (minimum version is now %(version)i)"
msgstr ""
"너무 오래되었으므로(현재 최소 버전은 %(version)i임) 서비스 내역에서 컴퓨트 "
"RPC 버전을 추출하는 데 실패"

#, python-format
msgid ""
"Failed to extract compute RPC version from service history for version "
"%(version)i"
msgstr "%(version)i 버전의 서비스 이력에서 컴퓨트 RPC 버전을 추출하는 데 실패"

msgid "Failed to fetch glance image"
msgstr "글랜스 이미지 페치 실패"

#, python-format
msgid "Failed to find an SR for volume %s"
msgstr "%s 볼륨의 SR을 찾는 데 실패"

#, python-format
msgid "Failed to forget the SR for volume %s"
msgstr "%s 볼륨의 SR을 무시(forget)하는 데 실패"

#, python-format
msgid "Failed to generate usage audit for instance on host %s"
msgstr "%s 호스트의 인스턴스에 대한 사용 감사를 생성하지 못했음"

#, python-format
msgid "Failed to get compute_info for %s"
msgstr "%s의 compute_info를 가져오는 데 실패"

msgid "Failed to get default networks"
msgstr "기본 네트워크를 가져오는 데 실패"

#, python-format
msgid ""
"Failed to get instance id for metadata request, provider %(provider)s "
"networks %(networks)s requester %(requester)s. Error: %(error)s"
msgstr ""
"메타데이터 요청의 인스턴스 ID, 제공자 %(provider)s 네트워크 %(networks)s 요청"
"자 %(requester)s을(를) 가져오는 데 실패했습니다. 오류: %(error)s"

#, python-format
msgid "Failed to get metadata for instance id: %s"
msgstr "인스턴스 ID에 대한 메타데이터를 가져오지 못했음: %s"

msgid "Failed to get nw_info"
msgstr "nw_info를 가져오지 못했음"

#, python-format
msgid "Failed to instantiate the download handler for %(scheme)s"
msgstr "%(scheme)s의 다운로드 핸들러를 인스턴스화하는 데 실패"

#, python-format
msgid ""
"Failed to kill process %(pid)s due to %(reason)s, while deleting the "
"instance."
msgstr ""
"인스턴스를 삭제하는 중에 %(reason)s(으)로 인해 %(pid)s 프로세스를 강제 종료하"
"는 데 실패했습니다."

#, python-format
msgid "Failed to load %(cfgfile)s: %(ex)s"
msgstr "%(cfgfile)s을(를) 로드하는 데 실패: %(ex)s"

#, python-format
msgid "Failed to load %s"
msgstr "%s을(를) 로드하는 데 실패"

#, python-format
msgid "Failed to load %s-api"
msgstr "%s-api를 로드하는 데 실패"

#, python-format
msgid ""
"Failed to mount container filesystem '%(image)s' on '%(target)s': %(errors)s"
msgstr ""
"컨테이너 파일 시스템 '%(image)s'을(를) '%(target)s'에 마운트하는 데 실패함: "
"%(errors)s"

msgid "Failed to notify cells of BDM destroy."
msgstr "셀에 BDM 삭제를 알리는 데 실패했습니다."

msgid "Failed to notify cells of BDM update/create."
msgstr "셀에 BDM 업데이트/생성을 알리는 데 실패했습니다."

msgid "Failed to notify cells of bw_usage update"
msgstr "셀에 bw_usage 업데이트를 알리지 못했음"

msgid "Failed to notify cells of instance fault"
msgstr "셀에 인스턴스 결함을 알리지 못했음"

msgid "Failed to notify cells of instance info cache update"
msgstr "셀에 인스턴스 정보 캐시 업데이트를 알리지 못했음"

#, python-format
msgid "Failed to remove %(base_file)s, error was %(error)s"
msgstr "%(base_file)s 제거 실패, 오류는 %(error)s입니다."

#, python-format
msgid "Failed to remove snapshot for VM %s"
msgstr "VM %s의 스냅샷을 제거하는 데 실패"

#, python-format
msgid "Failed to request Ironic to provision instance %(inst)s: %(reason)s"
msgstr "%(inst)s 인스턴스를 프로비저닝하기 위한 Ironic 요청 실패: %(reason)s"

#, python-format
msgid ""
"Failed to retrieve encryption metadata for volume %(volume_id)s: "
"%(exception)s"
msgstr ""
"%(volume_id)s 볼륨의 암호화 메타데이터를 검색하는 데 실패: %(exception)s"

msgid "Failed to retrieve ephemeral encryption key"
msgstr "임시 암호화 키를 검색하는 데 실패"

msgid "Failed to revert crashed migration"
msgstr "충돌한 마이그레이션을 되돌릴 수 없음"

#, python-format
msgid "Failed to roll back reservations %s"
msgstr "%s 예약을 철회하는 데 실패"

msgid "Failed to rollback quota for failed finish_resize"
msgstr "실패한 finish_resize의 할당량을 롤백하는 데 실패"

#, python-format
msgid "Failed to send message to cell: %(next_hop)s"
msgstr "셀에 메시지를 보내는 데 실패: %(next_hop)s"

msgid "Failed to send state update notification"
msgstr "상태 업데이트 알림을 발송하지 못했음"

msgid "Failed to send updated snapshot status to volume service."
msgstr "업데이트된 스냅샷 상태를 볼륨 서비스에 보내는 데 실패했습니다."

msgid "Failed to set instance name using multi_instance_display_name_template."
msgstr ""
"multi_instance_display_name_template를 사용하여 인스턴스 이름을 설정하는 데 "
"실패했습니다."

msgid "Failed to snapshot image"
msgstr "이미지의 스냅샷을 작성하는 데 실패"

#, python-format
msgid "Failed to start %(name)s on %(host)s:%(port)s with SSL support"
msgstr "SSL 지원을 갖는 %(host)s:%(port)s에서 %(name)s을(를) 시작하지 못했음"

msgid "Failed to start instance"
msgstr "인스턴스를 시작하는 데 실패"

msgid "Failed to stop instance"
msgstr "인스턴스를 중지하는 데 실패"

#, python-format
msgid "Failed to swap volume %(old_volume_id)s for %(new_volume_id)s"
msgstr "%(new_volume_id)s의 %(old_volume_id)s 볼륨을 스왑하는 데 실패"

msgid "Failed to teardown container filesystem"
msgstr "컨테이너 파일 시스템을 분해하는 데 실패"

msgid "Failed to umount container filesystem"
msgstr "컨테이너 파일 시스템을 umount하는 데 실패"

msgid "Failed to unpause instance"
msgstr "인스턴스의 일시정지를 취소하는 데 실패"

#, python-format
msgid "Failed to untar %(image_location)s to %(image_path)s"
msgstr ""
"%(image_location)s을(를) %(image_path)s(으)로 untar(압축 해제)하지 못했음"

msgid "Failed to update usages bulk deallocating floating IP"
msgstr "사용량 대량 할당 헤제 Floating IP를 업데이트하는 데 실패"

msgid "Failed to update usages deallocating fixed IP"
msgstr "사용량 업데이트에 실패하여 고정 IP를 할당 해제하는 중"

msgid "Failed to update usages deallocating floating IP"
msgstr "부동 IP 할당 해제 사용법을 업데이트하지 못했습니다. "

msgid "Failed to update usages deallocating network."
msgstr "네트워크 할당 해제 사용법을 업데이트하지 못했습니다. "

msgid "Failed to update usages deallocating security group"
msgstr "사용량 할당 해제 보안 그룹을 업데이트하지 못했음"

msgid "Failed to update usages deallocating server group"
msgstr "사용 할당 해제 서버 그룹을 업데이트하는 데 실패"

#, python-format
msgid "Failed to upload %(image_location)s to %(image_path)s"
msgstr "%(image_location)s을(를) %(image_path)s(으)로 업로드하지 못했음"

msgid "Failed to write configuration file"
msgstr "구성 파일을 쓰지 못했음"

msgid "Failed to write inbound.csr"
msgstr "inbound.csr을 작성하지 못했음"

msgid "Failed while plugging ib hostdev vif"
msgstr "ib hostdev vif를 플러그하는 중 실패"

msgid "Failed while plugging vif"
msgstr " vif를 플러그하는 중 실패"

msgid "Failed while unplugging ib hostdev vif"
msgstr "ib hostdev vif의 플러그를 해제하는 중 실패"

msgid "Failed while unplugging vif"
msgstr "vif 언플러그 중 실패"

msgid "Failure prepping block device"
msgstr "블록 디바이스 준비 실패"

msgid "Failure while cleaning up attached VDIs"
msgstr "첨부된 VDI 정리 중에 실패"

#, python-format
msgid "Fatal Exception running %(name)s %(type)s-hook: %(obj)s"
msgstr "%(name)s %(type)s-hook을(를) 실행하는 중 치명적 예외 발생: %(obj)s"

msgid "Guest does not have a console available"
msgstr "게스트에 사용 가능한 콘솔이 없음"

msgid "Host is member of a pool, but DB says otherwise"
msgstr "호스트가 풀의 멤버이지만, DB는 다르게 말합니다. "

#, python-format
msgid ""
"Host mismatch detected in stashed volume connector. Will use local volume "
"connector. Instance host is %(ihost)s. Local volume connector host is "
"%(chost)s. Stashed volume connector host is %(schost)s."
msgstr ""
"숨긴 볼륨 커넥터에서 호스트 불일치가 발견되었습니다. 로컬 볼륨 커넥터를 사용"
"합니다. 인스턴스 호스트가 %(ihost)s입니다. 로컬 볼륨 커넥터 호스트가 "
"%(chost)s입니다. 숨겨진 볼륨 커넥터 호스트는 %(schost)s입니다."

#, python-format
msgid ""
"Hostname has changed from %(old)s to %(new)s. A restart is required to take "
"effect."
msgstr ""
"호스트 이름이 %(old)s에서 %(new)s(으)로 변경되었습니다. 변경이 적용되려면 다"
"시 시작해야 합니다."

#, python-format
msgid "Image signature verification failed for image: %s"
msgstr "이미지의 이미지 서명 검증 실패: %s"

#, python-format
msgid "Image size %(size)d exceeded flavor allowed size %(allowed_size)d"
msgstr ""
"이미지 크기 %(size)d이(가) flavor 허용 크기 %(allowed_size)d을(를) 초과함"

#, python-format
msgid "Improper or unknown HTTP status code used: %d"
msgstr "올바르지 않거나 알 수 없는 HTTP 상태 코드 사용: %d"

#, python-format
msgid ""
"Initialize connection failed for volume %(vol)s on host %(host)s. Error: "
"%(msg)s Code: %(code)s. Attempting to terminate connection."
msgstr ""
"%(host)s 호스트에서 %(vol)s 볼륨의 연결을 초기화하는 데 실패했습니다. 오류: "
"%(msg)s 코드: %(code)s. 연결을 종료하려고 합니다."

#, python-format
msgid ""
"Instance compute service state on %s expected to be down, but it was up."
msgstr ""
" %s의 인스턴스 컴퓨트 서비스 상태가 작동 중단이어야 하는데 반대로 가동입니다."

msgid "Instance failed block device setup"
msgstr "인스턴스가 디바이스 설정을 차단하지 못했음"

#, python-format
msgid "Instance failed network setup after %(attempts)d attempt(s)"
msgstr "%(attempts)d번의 시도 후에 인스턴스가 네트워크를 설정하는 데 실패"

msgid "Instance failed to spawn"
msgstr "인스턴스가 파생하지 못했음"

#, python-format
msgid "Interface %s not found"
msgstr "%s 인터페이스를 찾을 수 없음"

#, python-format
msgid "Invalid server_string: %s"
msgstr "올바르지 않은 server_string: %s"

#, python-format
msgid "Live Migration failure: %s"
msgstr "라이브 마이그레이션 실패: %s"

msgid "Live migration failed."
msgstr "라이브 마이그레이션에 실패했습니다."

msgid "Migrate Receive failed"
msgstr "마이그레이션 수신 실패"

msgid "Migrate Send failed"
msgstr "마이그레이션 전송 실패"

#, python-format
msgid "Migration %s is not found during confirmation"
msgstr "확인 중에 %s 마이그레이션을 찾을 수 없음"

#, python-format
msgid "Migration for instance %(uuid)s refers to another host's instance!"
msgstr ""
"%(uuid)s 인스턴스의 마이그레이션에서 다른 호스트의 인스턴스를 참조합니다!"

#, python-format
msgid ""
"Migration of instance %(instance_id)s to host %(dest)s unexpectedly failed."
msgstr ""
"%(instance_id)s 인스턴스를 %(dest)s 호스트로 마이그레이션하는 작업이 예상치 "
"않게 실패했습니다."

msgid "Migration operation has aborted"
msgstr "마이그레이션 조작이 중단됨"

#, python-format
msgid "NBD error - previous umount did not cleanup /var/lock/qemu-nbd-%s."
msgstr ""
"NBD 오류 - 이전 umount를 통해 /var/lock/qemu-nbd-%s이(가) 정리되지 않았습니"
"다."

#, python-format
msgid ""
"NOT IMPLEMENTED: The call to %(method)s is not supported by the agent. args="
"%(args)r"
msgstr ""
"구현되지 않음: %(method)s에 대한 호출은 에이전트에서 지원하지 않습니다. args="
"%(args)r"

msgid "Network driver option required, but not specified"
msgstr "네트워크 드라이버 옵션이 필수이지만 지정되지 않았음"

#, python-format
msgid "Neutron Error adding rules to security group %s"
msgstr "보안 그룹 %s에 규칙을 추가하는 중 Neutron 오류 발생"

#, python-format
msgid "Neutron Error creating security group %s"
msgstr "보안 그룹 %s을(를) 생성하는 중 Neutron 오류 발생"

#, python-format
msgid "Neutron Error getting security group %s"
msgstr "보안 그룹 %s을(를) 가져오는 중 Neutron 오류 발생"

msgid "Neutron Error getting security groups"
msgstr "보안 그룹을 가져오는 중 Neutron 오류 발생"

#, python-format
msgid "Neutron Error unable to delete %s"
msgstr "%s을(를) 삭제할 수 없는 Neutron 오류"

#, python-format
msgid "Neutron Error updating security group %s"
msgstr "보안 그룹 %s을(를) 업데이트하는 중 Neutron 오류 발생"

msgid "Neutron Error:"
msgstr "Neutron 오류: "

#, python-format
msgid "Neutron Error: %s"
msgstr "Neutron 오류: %s"

#, python-format
msgid "Neutron Reported failure on event %(event)s for instance %(uuid)s"
msgstr "%(uuid)s 인스턴스의 %(event)s 이벤트에서 Neutron이 실패를 보고함"

#, python-format
msgid "Neutron error creating port on network %s"
msgstr "%s 네트워크에서 포트를 생성하는 중 Neutron 오류 발생"

#, python-format
msgid "No compute node record for host %s"
msgstr "%s 호스트의 컴퓨트 노드 레코드가 없음"

#, python-format
msgid "No load handler for %s"
msgstr "%s의 핸들러가 로드되지 않음"

#, python-format
msgid "No save handler for %s"
msgstr "%s의 핸들러를 저장하지 않음"

msgid "Not all properties needed are implemented in the compute driver"
msgstr "필요한 일부 특성이 컴퓨트 드라이버에 구현되지 않음"

msgid ""
"Periodic sync_power_state task had an error while processing an instance."
msgstr ""
"인스턴스를 처리하는 중에 정기적 sync_power_state 작업에 오류가 발생했습니다."

msgid "Periodic task failed to offload instance."
msgstr "정기적 작업을 통해 인스턴스를 오프로드하는 데 실패했습니다."

msgid "Plugin config_file get_val failed"
msgstr "config_file get_val 플러그인에 실패"

#, python-format
msgid "Pool-Join failed: %s"
msgstr "풀 결합 실패: %s"

#, python-format
msgid "Pool-eject failed: %s"
msgstr "풀 꺼내기 실패: %s"

#, python-format
msgid "Pool-set_name_label failed: %s"
msgstr "Pool-set_name_label 실패: %s"

#, python-format
msgid "Pre live migration failed at %s"
msgstr "사전 라이브 마이그레이션이 %s에서 실패함"

msgid "Read/Write data failed"
msgstr "데이터 읽기/쓰기 실패"

#, python-format
msgid "Requested VM Generation %s is not supported on this OS."
msgstr "요청된 VM 생성 %s은(는) 이 OS에서 지원되지 않습니다."

#, python-format
msgid "Requested VM Generation %s, but provided VHD instead of VHDX."
msgstr "VM 생성 %s이(가) 요청되었지만 VHDX 대신 VHD가 제공되었습니다."

#, python-format
msgid "Returning exception %s to caller"
msgstr "%s 예외를 호출자에게 리턴 중"

msgid "Service error occurred during cleanup_host"
msgstr "cleanup_host 중에 서비스 오류 발생"

msgid "Setting instance vm_state to ERROR"
msgstr "인스턴스 vm_state을(를) ERROR로 설정"

msgid "Shutting down VM (cleanly) failed."
msgstr "VM 종료(문제가 발생하지 않음)에 실패"

msgid "Shutting down VM (hard) failed"
msgstr "VM 종료(하드)에 실패"

msgid ""
"Something wrong happened when trying to delete snapshot from shelved "
"instance."
msgstr "shelved 인스턴스에서 스냅샷을 삭제할 때 문제가 발생했습니다."

#, python-format
msgid "TIMEOUT: The call to %(method)s timed out. args=%(args)r"
msgstr "TIMEOUT: %(method)s 호출이 제한시간 초과했습니다. args=%(args)r"

#, python-format
msgid "Temporary directory is invalid: %s"
msgstr "임시 디렉토리가 올바르지 않음: %s"

#, python-format
msgid "The Quobyte volume at %s is still in use."
msgstr "%s의 Quobyte 볼륨이 여전히 사용 중입니다."

#, python-format
msgid "The agent call to %(method)s returned an error: %(ret)r. args=%(args)r"
msgstr ""
"에이전트에서 %(method)s을(를) 호출하면 오류가 리턴됨: %(ret)r. args=%(args)r"

#, python-format
msgid ""
"The agent call to %(method)s returned an invalid response: %(ret)r. args="
"%(args)r"
msgstr ""
"에이전트에서 %(method)s을(를) 호출하면 올바르지 않은 응답이 리턴됨: %(ret)r. "
"args=%(args)r"

#, python-format
msgid "The call to %(method)s returned an error: %(e)s."
msgstr "%(method)s에 대한 호출이 오류를 리턴함: %(e)s."

#, python-format
msgid "The call to %(method)s returned an error: %(e)s. args=%(args)r"
msgstr "%(method)s 호출이 오류를 리턴했음: %(e)s. args=%(args)r"

msgid "Transfer data failed"
msgstr "데이터 전송 실패"

#, python-format
msgid "Unable to access floating IP %s"
msgstr "Floating IP %s에 액세스할 수 없음"

#, python-format
msgid "Unable to access floating IP for %s"
msgstr "%s의 Floating IP에 액세스할 수 없음"

msgid "Unable to access the rescue disk"
msgstr "복구 디스크에 액세스할 수 없음"

#, python-format
msgid "Unable to attach volume to instance %s"
msgstr "%s 인스턴스에 볼륨을 첨부할 수 없음"

#, python-format
msgid "Unable to clear device ID for port '%s'"
msgstr "'%s' 포트의 디바이스 ID를 지울 수 없음"

msgid "Unable to create VM snapshot, failing volume_snapshot operation."
msgstr "VM 스냅샷을 생성할 수 없으므로, volume_snapshot 작업에 실패합니다."

msgid ""
"Unable to create quiesced VM snapshot, attempting again with quiescing "
"disabled."
msgstr ""
"일시정지된 VM 스냅샷을 생성할 수 없으므로 일시정지를 비활성화하여 다시 시도합"
"니다."

msgid "Unable to destroy VBD"
msgstr "VBD를 영구 삭제할 수 없음"

#, python-format
msgid "Unable to execute %(cmd)s. Exception: %(exception)s"
msgstr "%(cmd)s을(를) 실행할 수 없습니다. 예외: %(exception)s"

msgid "Unable to find SR from VBD"
msgstr "VBD에서 SR을 찾을 수 없음"

msgid "Unable to find SR from VDI"
msgstr "VDI에서 SR을 찾을 수 없음"

msgid "Unable to get record of VDI"
msgstr "VDI 레코드를 가져올 수 없음"

msgid "Unable to get updated status"
msgstr "업데이트된 상태를 가져올 수 없음"

msgid "Unable to introduce VDI for SR"
msgstr "SR의 VDI를 도입할 수 없음"

msgid "Unable to introduce VDI on SR"
msgstr "SR에서 VDI를 도입할 수 없음"

msgid "Unable to load the virtualization driver"
msgstr "가상화 드라이버를 로드할 수 없음"

#, python-format
msgid "Unable to migrate VM %(vm_ref)s from %(host)s"
msgstr "%(host)s에서 VM %(vm_ref)s을(를) 마이그레이션할 수 없음"

#, python-format
msgid ""
"Unable to obtain RRD XML for VM %(vm_uuid)s with server details: %(server)s."
msgstr ""
"서버 세부사항을 사용하여 VM %(vm_uuid)s의 RRD XML을 얻을 수 없음: %(server)s."

#, python-format
msgid "Unable to parse rrd of %s"
msgstr "%s의 rrd를 구문 분석할 수 없음"

#, python-format
msgid "Unable to retrieve certificate with ID %(id)s: %(e)s"
msgstr "ID가 %(id)s인 인증서를 검색할 수 없음: %(e)s"

#, python-format
msgid "Unable to retrieve storage policy with name %s"
msgstr "%s 이름의 스토리지 정책을 검색할 수 없음"

#, python-format
msgid "Unable to set up pool: %s."
msgstr "풀을 설정할 수 없음: %s."

msgid "Unable to unplug VBD"
msgstr "VBD 플러그를 해제할 수 없음"

#, python-format
msgid "Unable to update instance VNIC index for port %s."
msgstr "%s 포트의 인스턴스 VNIC 색인을 업데이트할 수 없습니다."

msgid "Unexpected build failure, not rescheduling build."
msgstr "예상치 않은 빌드 실패 발생, 빌드를 재스케줄링하지 않음"

msgid "Unexpected error during post live migration at destination host."
msgstr ""
"대상 호스트에서 사후 라이브 마이그레이션을 수행하는 중에 예상치 못한 오류가 "
"발생했습니다."

#, python-format
msgid "Unexpected error while checking process %(pid)s."
msgstr "%(pid)s 프로세스를 확인하는 중에 예상치 못한 오류가 발생했습니다."

msgid "Unexpected error while reporting service status"
msgstr "서비스 상태를 보고하는 중에 예상치 못한 오류 발생"

msgid "Unexpected exception in API method"
msgstr "API 메소드에서 예상치 못한 예외 발생"

#, python-format
msgid "Unknown cell '%(cell_name)s' when trying to update capabilities"
msgstr "기능을 업데이트하려는 중에 알 수 없는 셀 '%(cell_name)s'이(가) 있음"

#, python-format
msgid "Unknown cell '%(cell_name)s' when trying to update capacities"
msgstr "용량을 업데이트하려는 중에 알 수 없는 셀 '%(cell_name)s'이(가) 있음"

#, python-format
msgid "Unknown content in connection_info/access_mode: %s"
msgstr "connection_info/access_mode의 알 수 없는 콘텐츠: %s"

msgid "Unshelve attempted but an error has occurred"
msgstr "Unshelve를 시도했지만 오류 발생"

msgid "Unshelve attempted but vm_state not SHELVED or SHELVED_OFFLOADED"
msgstr "Unshelve를 시도했지만 vm_state가 SHELVED 또는 SHELVED_OFFLOADED가 아님"

msgid "Vifs plug failed"
msgstr "Vifs 플러그에 실패"

#, python-format
msgid "Volume %s is associated with the instance but no SR was found for it"
msgstr "%s 볼륨이 인스턴스와 연결되었지만 SR을 찾지 못함"

#, python-format
msgid "Volume is not writable. Please broaden the file permissions. Mount: %s"
msgstr "볼륨에 쓸 수 없습니다. 파일 권한을 확장하십시오. 마운트: %s"

#, python-format
msgid ""
"When loading the module %(module_str)s the following error occurred: %(ex)s"
msgstr "%(module_str)s 모듈을 로드할 때 다음 오류 발생: %(ex)s"

msgid ""
"XenAPI is unable to find a Storage Repository to install guest instances on. "
"Please check your configuration (e.g. set a default SR for the pool) and/or "
"configure the flag 'sr_matching_filter'."
msgstr ""
"XenAPI에서 게스트 인스턴스를 설치할 스토리지 저장소를 찾을 수 없습니다.  구성"
"(예: 풀의 기본 SR 설정)을 확인하고/하거나 'sr_matching_filter' 플래그를 구성"
"하십시오."

msgid ""
"You are running nova-compute on an unsupported version of Windows (older "
"than Windows / Hyper-V Server 2012). The support for this version of Windows "
"has been removed in Mitaka."
msgstr ""
"지원되지 않은 버전의 Windows(Windows / Hyper-V Server 2012 이전 버전)에서 "
"nova-compute를 실행 중입니다. 이 버전의 Windows 지원은 Mitaka에서 제거되었습"
"니다."

msgid "_migrate_disk_resizing_down failed. Restoring orig vm"
msgstr ""
"_migrate_disk_resizing_down에 실패했습니다. 원본 vm을 복원하는 중입니다."

#, python-format
msgid "_migrate_disk_resizing_up failed. Restoring orig vm due_to: %s."
msgstr ""
"_migrate_disk_resizing_up에 실패했습니다. 다음으로 인해 원본 vm을 복원 중입니"
"다. %s."

#, python-format
msgid "allocate_port_for_instance returned %(ports)s ports"
msgstr "allocate_port_for_instance에서 %(ports)s 포트를 반환함"

msgid "attaching network adapter failed."
msgstr "네트워크 어댑터 첨부에 실패했습니다."

msgid "detaching network adapter failed."
msgstr "네트워크 어댑터 분리에 실패했습니다."

msgid "error during stop() in sync_power_state."
msgstr "sync_power_state에서 stop() 중에 오류 발생"

#, python-format
msgid "error opening rbd image %s"
msgstr "rbd 이미지 %s 열기 오류"

#, python-format
msgid "image %(volume)s in pool %(pool)s has snapshots, failed to remove"
msgstr ""
"%(pool)s 풀의 %(volume)s 이미지에 스냅샷이 있으므로, 제거하는 데 실패함"

#, python-format
msgid "kill -HUP dnsmasq threw %s"
msgstr "kill -HUP dnsmasq에서 %s 발생"

#, python-format
msgid "killing radvd threw %s"
msgstr "radvd 강제 종료가 %s을(를) 처리함"

msgid "nbd module not loaded"
msgstr "nbd 모듈이 로드되지 않았음"

msgid "set_admin_password failed"
msgstr "set_admin_password 실패"

#, python-format
msgid ""
"status is UNKNOWN from vm_state=%(vm_state)s task_state=%(task_state)s. Bad "
"upgrade or db corrupted?"
msgstr ""
"vm_state=%(vm_state)s task_state=%(task_state)s의 상태가 알 수 없음입니다. 업"
"그레이드 불량이거나 db가 손상되었습니다."
